tests/func/test_add.py

```diff
@@ -216,7 +216,7 @@ def test(self):
         self.assertEqual(ret, 0)
 
         d = load_stage_file("bar.dvc")
-        self.assertEqual(d["outs"][0]["path"], bar)
+        self.assertEqual(d["outs"][0]["path"], self.BAR)
```

Comment:
@casperdcl Please avoid modifying unrelated things :slightly_smiling_face:

Reply:
Totally not unrelated, this is a test fix. Fails otherwise.

Reply:
@casperdcl Oh, totally mised that one. Right, `bar` is an abs path, makes sense now. Thanks for clarifying!